modules/apps/translation/translation-web/src/main/resources/META-INF/resources/js/translate/components/TranslateFieldSetEntries.tsx

```diff
@@ -128,28 +219,51 @@ const TranslateFieldEditor = ({
 				<ClayForm.Group>
 					<label className="control-label">{label}</label>
 
-					<ClassicEditor
-						editorConfig={{
-							...editorConfiguration.editorConfig,
-							contentsLangDirection: targetContentDir,
-						}}
-						name={id}
-						onChange={(data) => {
-							setContent(data);
-							onChange(data);
-							internalUpdateRef.current = true;
-						}}
-						onInstanceReady={({editor}) => {
-
-							// LPS-139363
-
-							editor?.setData?.(content, {
-								interal: true,
-								noSnapshot: true,
-							});
-						}}
-						ref={editorRef}
-					/>
+					{Liferay.FeatureFlags['LPD-11235'] ? (
+						<CKEditor5ClassicEditor
+							config={{
+								...editorConfiguration.editorConfig,
+								initialData: targetContent,
+								language: {
+									content:
+										getISO639LanguageCode(targetLanguageId),
+								},
+							}}
+							onChange={(event, editor) => {
+								const data = editor.getData();
+
+								setContent(data);
+
+								onChange(data);
+							}}
+							onReady={onReady}
+						/>
+					) : (
+						<ClassicEditor
+							editorConfig={{
+								...editorConfiguration.editorConfig,
+								contentsLangDirection: targetContentDir,
+							}}
+							name={id}
+							onChange={(data: string) => {
+								setContent(data);
```

Comment:
Minor suggestion. I would create a `handleOnChange()` function that calls `setContent(data); onChange(data);`. This is used in onChange from `ClassicEditor` and `CKEditor5ClassicEditor`, but also in the source edition mode.